contributors/guide/github-workflow.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "GitHub Workflow"
-weight: 6
-description: |
+Title: "GitHub Workflow"
+Weight: 6
+Description: |
   An overview of the GitHub workflow used by the Kubernetes project. It includes
   some tips and suggestions on things such as keeping your local environment in
   sync with upstream and commit hygiene.
 ---
 
 ![Git workflow](git_workflow.png)
 
-### 1 Fork in the cloud
+### 1 - Fork in the cloud
```

Comment:
Any reason why these hyphens are required here (and below)?

Reply:
Just to make it easier to understand the steps